landing-page/index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+   <title>CtD class-Exerc.3-Landing page</title>
+   <link rel="stylesheet" type="text/css" href="styles.css">
+</head>
+
+<body>
+  <header id="showcase">
+    <h1>Welcome!</h1>
+    <h1 id="simbol">Gizmatic</h1>
+        <p>Founded in 1901 Gizmatic has been making the world's best gizmos.</p> 
+    <a href="www.gizmatic.com/about" class="button1">Read More</a> 
+    
+    <section id="section-a">
+    <a class="button2"
+       href="https://www.gizmatic.com/customer&service">Service</a>
```

Comment:
Same thing about indentation. Elements that are supposed to be contained within a parent element, like these links inside a section, should be indented in 2 spaces from their enclosing parent

Reply:
Another thing here: having the html attributes on a new line ( here, `href` below the `class` attribute) in regular html is unconventional. Browsers are pretty forgiving with things like this, but it's important to follow conventions that the human readers of your code expect. If you're concerned about line length you could do something like this: 
```
<a class="button2" href="https://www.gizmatic.com/customer&service">
  Service
</a>
```